metadata-ingestion/src/datahub/ingestion/source/csv_enricher.py

```diff
@@ -106,17 +106,19 @@ class CSVEnricherSource(Source):
     The format of the CSV is demonstrated below. The header is required and URNs should be surrounded by quotes when they contains commas (most URNs contains commas).
 
     ```
-    resource,subresource,glossary_terms,tags,owners,ownership_type,description,domain
-    "urn:li:dataset:(urn:li:dataPlatform:snowflake,datahub.growth.users,PROD)",,[urn:li:glossaryTerm:Users],[urn:li:tag:HighQuality],[urn:li:corpuser:lfoe|urn:li:corpuser:jdoe],TECHNICAL_OWNER,"description for users table",urn:li:domain:Engineering
-    "urn:li:dataset:(urn:li:dataPlatform:hive,datahub.growth.users,PROD)",first_name,[urn:li:glossaryTerm:FirstName],,,,"first_name description"
-    "urn:li:dataset:(urn:li:dataPlatform:hive,datahub.growth.users,PROD)",last_name,[urn:li:glossaryTerm:LastName],,,,"last_name description"
+    resource,subresource,glossary_terms,tags,owners,ownership_type,description,domain,ownership_type_urn
+    "urn:li:dataset:(urn:li:dataPlatform:snowflake,datahub.growth.users,PROD)",,[urn:li:glossaryTerm:Users],[urn:li:tag:HighQuality],[urn:li:corpuser:lfoe|urn:li:corpuser:jdoe],CUSTOM,"description for users table",urn:li:domain:Engineering,urn:li:ownershipType:a0e9176c-d8cf-4b11-963b-f7a1bc2333c9
```

Comment:
This urn: `urn:li:ownershipType:a0e9176c-d8cf-4b11-963b-f7a1bc2333c9` is not the urn for technical owner, that should be: https://github.com/datahub-project/datahub/blob/7ee86bbb84090a6a8b0e7ffd115b9a3b6eeac085/metadata-service/war/src/main/resources/boot/ownership_types.json#L3

Reply:
I am not trying this to be an example for Technical owner. That is already present